src/glossary.md

```diff
@@ -0,0 +1,281 @@
+# Fuel Glossary
+
+## Address
+
+An address is a cryptographic hash representing an identity of either a wallet, contract address, or a predicate root.
+
+## AssetId
+
+An AssetId is a unique address for an on-chain asset. It is defined as the root of the bytecode of the contract minting the asset.
```

Comment:
```suggestion
An AssetId is a unique identifier for an on-chain asset. It is defined as the root of the bytecode of the contract minting the asset.
```
Generally, we try to maintain the distinction between `ContractId`s and `Address`es  : they both wrap a `b256` but this abstraction (implemented in the std-lib) makes interactions with the two type-safe.